app/config/config.json

```diff
@@ -0,0 +1,22 @@
+{
+  "restApiRoot": "/api",
+  "host": "127.0.0.1",
+  "port": 8888,
+  "remoting": {
+    "context": false,
+    "rest": {
+      "handleErrors": false,
+      "normalizeHttpPath": false,
+      "xml": false
+    },
+    "json": {
+      "strict": false,
+      "limit": "100kb"
+    },
+    "urlencoded": {
+      "extended": true,
+      "limit": "100kb"
+    },
+    "cors": false
```

Comment:
could be pretty important for microservices that are addresses under different domains